### DISCUSSION TITLE
ObservableHQ

### DISCUSSION THREAD

**Discussion by cornhundred** [visgl/deck.gl#6251] [General]
Hi, @ibgreen I mentioned in the UCA-TAC meeting that I started putting together a collection of deck.gl examples on ObservableHQ. 

https://observablehq.com/collection/@cornhundred/deck-gl

ObservableHQ is a service that lets you make notebooks in JavaScript and seems to work well with deck.gl. It might be good to set up a collection of notebooks that goes through the deck.gl examples on this service.

**Comment by ibgreen** [comment]
@cornhundred This looks like a great initiative. I think it would be great to link to these from the [learning resources](https://deck.gl/docs/get-started/learning-resources) page on the deck.gl website. I think you could open a PR adding that link. 

I note that @Pessimistress already has an observable page link with a bunch of examples.  Perhaps we should have a visgl account on Observable and collect them all there?

**Reply by cornhundred** [reply]
Hi @ibgreen, sure that sounds great - I can make a pull request. Yes, I had seen many of @Pessimistress's examples on ObservableHQ too - https://observablehq.com/collection/@pessimistress/deck-gl-tutorials. I would think maybe making a visgl account on Observable would be the way to go. This account could then collect these notebooks from different collections and add new ones as needed.
